public/scripts/main.js

```diff
@@ -244,9 +246,12 @@ const tooltipBox = new Konva.Rect({
   cornerRadius: 10,
 });
 
+const playerInventory = new Inventory();
```

Comment:
I'm thinking if an instance of `inventory` should be created in `Game` along with the player given that it should be persistent? Thoughts? @matnguyen @Bruno-DaSilva @abhatthal @Mibrer